src/Chapter1.hs

```diff
@@ -568,7 +578,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' || c == 'y' = True
+  | otherwise = False
```

Comment:
If you write it with `||`, I guess there is no much benefits of guards. You can:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' || c == 'y' 
```

We recommended guards, if you would check each case separately:

```haskell
  | c == 'a'  = True
  | c == 'e' = True
  | ...
```

Reply:
Thanks. Will do.